### PR TITLE
[Form] Mentioned the change in the "preferred choices" behavior

### LINKED ISSUE #12023: [Form] repeat preferred choices in list of all choices
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#32658
| PR author(s) | @Seb33300, @xabbuh
| Merged in    | 4.4

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #12023.

### CODE REVIEW

**Comment by xabbuh** [comment]
The description of this option starts with "This option allows you to move certain choices [...]". Should we reword it to replace "move" with something like "[...] allows you to render certain choices [...]"?

**Comment by javiereguiluz** [comment]
You are right! I've made more changes. Thanks.

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
👍
